### ISSUE TITLE
Cannot delegate to generic method

### ISSUE THREAD

**Issue by bugadani** [Kobzol/rust-delegate#75]
The following is I believe not expanded correctly:

```rust
struct Foo;
impl Foo {
  fn foo<P>(&self) { todo!() }
}

struct Bar(Foo);
impl Bar {
  delegate! {
    to &self.0 {
      fn foo<P>(&self);
    }
  }
}
```

The macro output, according to rust-analyzer is the following:

```rust
#[inline]
fn foo<P>(&self) {
    &self.0.foo(); // instead of .foo::<P>();
}
```

**Comment by Kobzol** [comment]
Hi, thanks for the report, good point. Could you please try if [this PR](https://github.com/Kobzol/rust-delegate/pull/76) works for you?
```toml
[dependencies]
delegate = { git = "https://github.com/kobzol/rust-delegate", branch = "delegate-generic-args" }
```

**Comment by bugadani** [comment]
Thank you for your quick handling of this issue, the PR works for me.

**Comment by Kobzol** [comment]
Awesome. I will merge it and release a new version in the near future.

**Comment by Kobzol** [comment]
Published v0.13.1 with the fix.
